acra/regulations/_index.md

```diff
@@ -2,3 +2,50 @@
+
+Acra is an encryption tool that allows processing sensitive data with great care and secureness — namely, encrypting the data in a certain way. Acra performs encryption through [Themis]({{< ref "/themis/" >}}) in [Secure Cell]({{< ref "themis/crypto-theory/cryptosystems/secure-cell.md" >}})/Seal mode (AES256 + GCM). This corresponds to the &quot;state of the art&quot; (as mentioned in the [full text of the GDPR](https://gdpr-info.eu/) regulation) algorithms of symmetric encryption. Also, Acra is securely pre-configured by default and carries out extensive logging of all its actions &quot;out of the box&quot;. Using Acra helps reaching compliance with some of the demands of the articles 32 ([Security of processing](https://gdpr-info.eu/art-32-gdpr/)) and 25 ([Data protection by design and by default](https://gdpr-info.eu/art-25-gdpr/)) of GDPR, as well as helps comply with the articles 33 and 34  ([Notification of a personal data breach to the supervisory authority](http://gdpr-info.eu/art-33-gdpr/) and [Communication of a personal data breach to the data subject](http://gdpr-info.eu/art-34-gdpr/)) of GDPR.
+
+## Security of processing
+[Article 32 of GDPR](https://gdpr-info.eu/art-32-gdpr/)
+
+Acra provides &quot;state of the art security of processing&quot; required in article 32 of GDPR through providing data encryption and integrity check of the encrypted data.
+
+_&gt; &quot;Taking into account the state of the art, the costs of implementation and the nature, scope, context and purposes of processing as well as the risk of varying likelihood and severity for the rights and freedoms of natural persons, the controller and the processor shall implement appropriate technical and organisational measures to ensure a level of security appropriate to the risk,...&quot;_
+
+Also, the components of Acra allow encrypting data on the client&#39;s side, which enables secure transfer of the data in encrypted form through an untrusted channel.
+
+## Data protection by design and by default
+
+[Article 25 of GDPR](https://gdpr-info.eu/art-25-gdpr/)
+
+The default settings of Acra are pre-configured in the most secure way to provide that exact &quot;secure by default&quot; state of the system. This is enabled through the default encryption settings for the data transfer between the Acra&#39;s components and the database ( [SSL](http://info.ssl.com/article.aspx?id=10241) and [Secure Session]({{< ref "themis/crypto-theory/cryptosystems/secure-session.md" >}}) are used).
+
+_&gt;&quot;In particular, such measures shall ensure that by default personal data are not made accessible without the individual&#39;s intervention to an indefinite number of natural persons.&quot;_
+
+## Logging, Intrusion Detection, and Notification
+
+[Articles 33](https://gdpr-info.eu/art-33-gdpr/) [and 34 of GDPR](https://gdpr-info.eu/art-34-gdpr/)
+
+Acra makes the communication with the users in case of a data breach easier because the data is encrypted (see Art 34 of GDPR):
```

Comment:
- Made grammatical structure less convoluted.
- I don't think you need to refer to Art. 34 again, it's already linked above.

```suggestion
Acra encrypts sensitive data, making it easier to communicate with the users in case of a data breach:
```